tests/unit/models/cart-model-test.js

```diff
@@ -312,8 +314,15 @@ test('it detects google analytics and appends the cross-domain linker param', fu
   assert.equal(model.checkoutUrl, `${baseUrl}/${variantId}:${quantity}?${query}`);
 
   const variant = singleProductFixture.product_listing.variants[1];
+  let globalScope;
```

Comment:
yeah, i thought about that, i wasn't sure if it was ideal to include that since it's not being tested so i was waiting for feedback. I guess i'd just include it  already.

Thanks